conceptnet5/vectors/evaluation/wordsim.py

```diff
@@ -311,7 +311,7 @@ def compute_semeval_score(pearson_score, spearman_score):
     Take a harmonic mean of a Pearson correlation coefficient and a Spearman correlation
     coefficient.
     """
-    if any(np.isnan(x) for x in [spearman_score['acc'], pearson_score['acc']]):
+    if any(np.isnan(x) or x <=0 for x in [spearman_score['low'], pearson_score['low']]):
```

Comment:
Hmm. This seems a bit inconsistent with the SemEval score: if your correlations were positive but insignificant, you would still get a SemEval score that's positive (and also insignificant). And if your correlations have data that can be measured, but one of your correlations was zero or negative, you get a SemEval score of 0 (different from `NaN`).

I'd like to reserve the result of `NaN` to mean "this data can't be evaluated with this evaluation".